### ISSUE TITLE
Is there autoupdate procedure?

### ISSUE THREAD

**Issue by den-vasyliev** [datreeio/CRDs-catalog#129]
Hello!

Flux source refreshing so frequently - is there a chance to get fresh updates here?

Thanks for the catalogue!

**Comment by eyarz** [comment]
Unfortunately, there is no auto-update, so any required update is done manually.
We'll need the help of the community to design and coding capability.

**Comment by den-vasyliev** [comment]
So, to update flux crd's it's needs to open PR for the exact release - am I right?
Beleive we can create such step in GitHubflow to get refresh periodically

**Comment by den-vasyliev** [comment]
thanks!

**Comment by xNok** [comment]
I was curious about what the `index.yaml` actually does. This seems like a good place to build a script that checks for updates.

**Comment by seanpoulter-foci** [comment]
> Unfortunately, there is no auto-update, so any required update is done manually (by submitting a PR). We'll need the help of the community to design and coding capability.

Would you want to focus on automating the process? It seems like we're building up a queue of PRs. I'd be happy to facilitate an ensemble (mob) for everyone interested to get'r done.

**Comment by apresti12** [comment]
Has there been any progress on this?  I think you could just have a map of crds to urls

``` yaml
external-secrets.io: 
  - https://raw.githubusercontent.com/external-secrets/external-secrets/main/config/crds/bases/external-secrets.io_clusterexternalsecrets.yaml
```

Then just run a github action on a schedule to pull the latest CRDs and use a conversion script like [this](https://github.com/yannh/kubeconform/blob/master/scripts/openapi2jsonschema.py) from the kubeconform repo

**Comment by eyarz** [comment]
@apresti12 totally open to get any help on this :)

**Comment by seanpoulter** [comment]
> @apresti12 totally open to get any help on this :)

You want to auto-update the catalog?

**Comment by eyarz** [comment]
> You want a PR to auto-update the catalog?

yes, the idea is that if the CRD schema was updated on the source it should somehow update the schema in this repo.
